gsd-schema/validation/schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GSD Identifier",
+  "description": "A schema for describing a GSD identifier.",
+  "type": "object",
+  "properties": {
+    "schema_format": {
+      "type": "string"
+    },
+    "schema_version": {
+
+    },
+    "osv": {
```

Comment:
This should be "OSV"

Reply:
Per https://github.com/cloudsecurityalliance/gsd-project/issues/29, we're moving towards camelCase for keys. Is there a particular consideration for why OSV should be capitalized?

Because this is essentially the v1 schema, we will be re-parsing the existing data to ensure consistency, and the existing entries using uppercase will be updated as we enable the schema enforcement.